ProcessList.c

```diff
@@ -170,73 +320,96 @@ static long ProcessList_treeProcessCompare(const void* v1, const void* v2) {
    const Process *p1 = (const Process*)v1;
    const Process *p2 = (const Process*)v2;
 
-   return p1->pid - p2->pid;
+   // return p1->tree_left - p2->tree_left;
+   return (p1->tree_left < p2->tree_left ? -1 : 1);
```

Comment:
`SPACESHIP_NUMBER()` ?

Reply:
What is your proposal actually is?
Sorry for my lack of knowledge.

Reply:
There is a helper macro https://github.com/htop-dev/htop/blob/95f553b10c3276c731bb7e3d9a917e9d98322e85/Macros.h#L23 which does a [three way comparison](https://en.wikipedia.org/wiki/Three-way_comparison) on numbers.